src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -75,6 +76,7 @@
 
 import static org.thoughtcrime.securesms.util.RelayUtil.REQUEST_RELAY;
 import static org.thoughtcrime.securesms.util.RelayUtil.acquireRelayMessageContent;
+import static org.thoughtcrime.securesms.util.RelayUtil.isForwarding;
```

Comment:
at a first glance this looks like an unnecessary import

Reply:
I optimized the imports.